zt_backend/models/generate_schema.py

```diff
@@ -62,4 +63,5 @@ def generate_schema():
     generate_json(Text,'text')
     generate_json(Timer,'timer')
     generate_json(iFrame,'iframe')
-    generate_json(HTML,'html')
\ No newline at end of file
+    generate_json(HTML,'html')
+    generate_json(ListComponent,'list')
```

Comment:
I see this is only the one component. I imagine everything was working without including the others

Reply:
@HarshitaDPoojary @metc1999

Reply:
Working even after removing the method.
Removed in [today's commit](https://github.com/Zero-True/zero-true/pull/277/commits/644eb127e48c34edf17c1178188def6aa83e650d#diff-8f6a2950b80f56eb4fa38d625ddc00131c08c7698d77abc3500252a80d528722L66)